### PR TITLE
[Update] - styles in the hero section

### PR DESCRIPTION
**Pull request by will0101**
Refactored the hero section to use Flexbox

### CODE REVIEW

File: src/css/modules/pages/home.css [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   font-size: var(--font-size-6);
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
 }
 
-.home-model .content-block__intro,
-.home-portfolio .content-block__intro {
-  margin-left: auto;
+/* Should I be using em or px? */
```
Comment (tylergaw) [inline src/css/modules/pages/home.css:9]: @will0101 I always us `em` for media queries. I do that based on the findings in this article https://zellwk.com/blog/media-query-units/

File: src/css/modules/pages/home.css [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+@media (max-width: 990px) {
```
Comment (tylergaw) [inline src/css/modules/pages/home.css:10]: We likely want a `min-width` here and for most media queries. That way we're saying that small screens are the baseline styles and only change styles if the screen reaches a certain width. That's a mobile-first approach that I try to follow. Effectively they do the same thing, but just a shift in thinking to where we're layering on style and functionality as more screen is available

File: src/css/modules/pages/home.css (hunk 1 above)
Comment (will0101) [reply src/css/modules/pages/home.css:9]: There's a lot of great stuff in this article. I just bookmarked it for future reference.

File: src/css/modules/pages/home.css (hunk 2 above)
Comment (will0101) [reply src/css/modules/pages/home.css:10]: Mobile-first and onward got it 👍.

**Pull request closed without merging**